alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php

```diff
@@ -123,20 +123,20 @@ protected function getBroadcastUrl(LiveStreamEntry $entry, $protocol, $hostname,
 		
 		$url = "$protocol://$hostname";
 		
-		$params = array(
-			'p' => $this->partnerId,
-			'e' => $entry->getId(),
-			'i' => $mediaServerIndex,
-			't' => $entry->getStreamPassword(),
-		);
-		$paramsStr = http_build_query($params);
-		
-		$streamName = '';
-		if($concatStreamName)
+		$params = array();
+		//Support eCDN partner using old live servers that mush recieve additional info to operate
+		if(PermissionPeer::isValidForPartner("FEATURE_HYBRID_ECDN", $entry->getPartnerId()))
 		{
-			$streamName = $entry->getId() . '_%i';
+			$params = array(
+					'p' => $this->partnerId,
+					'e' => $entry->getId(),
+					'i' => $mediaServerIndex,
+			);
 		}
+		$params = array_merge($params, array('t' => $entry->getStreamPassword()));
+		$paramsStr = http_build_query($params);
 		
+		$url .= $concatStreamName ? "/" . $entry->getId() . '_%i' : '';
 		return "$url/$streamName?$paramsStr"; 
```

Comment:
You are right I used $url .= instead of defining and using that variable 

Fixed please look again